### ISSUE TITLE
Inner and outer polygons in the radar boxplot may overlap.

### ISSUE THREAD

**Issue by yiyuanzou** [caiohamamura/radarBoxplot-R#1]
When there are only few data points, the inner and outer polygons may overlap.
This is because q100 (Line 123) may be less than q75 after removing outliers.
Perhaps add the following code after Line 123:

```
q100[q100 < q75] <- q75[q100 < q75]
q0[q0 > q25] <- q25[q0 > q25]
```

**Comment by caiohamamura** [comment]
Hello @yiyuanzou, thank you for using my package, great to know people are using it. Can you provide me an example dataset where this happens so I can test here?

**Comment by yiyuanzou** [comment]
Hi @caiohamamura, thank you for sharing the code for drawing radar boxplots. It looks really interesting. Maybe you can use the following code for testing.

```
library(radarBoxplot)

ID <- c("sample1", "sample2")
random1 <- rexp(8, 1/50)
random2 <- rexp(8, 1/50)
random3 <- rexp(8, 1/50)
random4 <- rexp(8, 1/50)
random5 <- rexp(8, 1/50)
random6 <- rexp(8, 1/50)

df <- data.frame(ID, random1, random2, random3, random4, random5, random6)
radarBoxplot(ID ~ ., df)
```

**Comment by caiohamamura** [comment]
Yeah, you are right @yiyuanzou , can you make a pull request? Do you know how? So that your name will be here as a contributor. You will need to create your own fork of the repository, edit the relevant part. Then I think in the main page of your fork the button Pull Request will appear, something like this:

![image](https://github.com/user-attachments/assets/0937340d-447e-4e6e-a3bb-164b8be9608f)

**Comment by yiyuanzou** [comment]
Hi @caiohamamura, thank you for your guidance. I have made a pull request. Please check if everything is okay :)

**Comment by caiohamamura** [comment]
Merged pull request https://github.com/caiohamamura/radarBoxplot-R/pull/2
